ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-19.yaml

```diff
@@ -0,0 +1,174 @@
+base_images:
+  cli:
+    name: "4.19"
+    namespace: ocp
+    tag: cli
+  ocp-4:
+    name: automation-flavors
+    namespace: stackrox
+    tag: openshift-4-stable
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: apollo-ci
+    namespace: stackrox
+    tag: stackrox-ui-test-0.4.8
```

Comment:
Why 4.8 while we are already on 4.9?

- https://github.com/openshift/release/pull/65929